<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NumberBasesWidget</class>
 <widget class="QWidget" name="NumberBasesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>885</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Decimal</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="decimal"/>
       </item>
       <item>
        <widget class="QPushButton" name="copyDecimalButton">
         <property name="toolTip">
          <string>Copy</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::EditCopy"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pasteDecimalButton">
         <property name="toolTip">
          <string>Paste</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::EditPaste"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearDecimalButton">
         <property name="toolTip">
          <string>Clear</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::EditClear"/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Binary</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="binary"/>
       </item>
       <item>
        <widget class="QPushButton" name="copyBinaryButton">
         <property name="toolTip">
          <string>Copy</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::EditCopy"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pasteBinaryButton">
         <property name="toolTip">
          <string>Paste</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::EditPaste"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="claerBinaryButton">
         <property name="toolTip">
          <string>Clear</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::EditClear"/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Octal</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="octal"/>
       </item>
       <item>
        <widget class="QPushButton" name="copyOctalButton">
         <property name="toolTip">
          <string>Copy</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::EditCopy"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pasteOctalButton">
         <property name="toolTip">
          <string>Paste</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::EditPaste"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearOctalButton">
         <property name="toolTip">
          <string>Clear</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::EditClear"/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Hexadecimal</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLineEdit" name="hexadecimal"/>
       </item>
       <item>
        <widget class="QPushButton" name="copyHexButton">
         <property name="toolTip">
          <string>Copy</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::EditCopy"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pasteHexButton">
         <property name="toolTip">
          <string>Paste</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::EditPaste"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearHexButton">
         <property name="toolTip">
          <string>Clear</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::EditClear"/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>ASCII</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="2">
        <widget class="QPushButton" name="pasteASCIIButton">
         <property name="toolTip">
          <string>Paste</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::EditPaste"/>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLineEdit" name="ascii"/>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="clearASCIIButton">
         <property name="toolTip">
          <string>Clear</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::EditClear"/>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="copyASCIIButton">
         <property name="toolTip">
          <string>Copy</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::EditCopy"/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>UTF-8</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLineEdit" name="utf8"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_18">
         <property name="toolTip">
          <string>Copy</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::EditCopy"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_17">
         <property name="toolTip">
          <string>Paste</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::EditPaste"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_16">
         <property name="toolTip">
          <string>Clear</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::EditClear"/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
